recipes/backoff/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "backoff" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "330e2cee2525ab49c8bf7d6ec7132635479ecb164c9cc69064c8b37715ff0fad" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - backoff
+
+about:
+  home: https://github.com/litl/backoff
+  license: MIT
+  license_family: MIT
```

Comment:
I added the LICENSE to the manifest, and just released backoff 1.4.3 which now includes the license file in the sdist tarball.